resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+
+![](/resources_pages/imgs/jupyter_lab_r_kernel.png)
+
+Sometimes a kernel loads, but doesn't work as expected. To test whether your installation was done correctly now type `library(tidyverse)` in the code cell and click on the run button to run the cell. If your R kernel works you should see something like the image below:
+
+![](/resources_pages/imgs/jupyter_lab_r_kernel2.png)
 
 ## LaTeX
 
-Download the installer from [http://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe](http://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe). After downloading run the `install-tl-windows.exe` file and follow the installation instructions choosing the default options.
+We will install the lightest possible version of LaTeX and it's necessary packages as possible so that we can render Jupyter notebooks and R Markdown documents to html and PDF. If you have previously installed LaTeX, please uninstall it before proceeding with these instructions.
 
-*note - if a screen appears titled "Windows protected your PC", click on "More info" link and the "Run anyway" button*
+First, open RStudio and run the following commands to install the `tinytex` package and setup `tinytex`:
 
-After installation, restart Powershell and type the following to ask for the version:
+```
+install.packages('tinytex')
+tinytex::install_tinytex()
+```
+
+Note that you might see two error messages regarding lua during the installation, you can safely ignore these, the installation will complete successfully after clicking "OK".
+
+
+You can check that the installation is working by typing the following in a terminal to ask for the version of latex:
 
 ```
 latex --version
 ```
 
-you should see something like this if you were successful:
+You should see something like this if you were successful:
 
 ```
-pdfTeX 3.14159265-2.6-1.40.19 (TeX Live 2018/W32TeX)
-kpathsea version 6.3.0
-Copyright 2018 Han The Thanh (pdfTeX) et al.
+pdfTeX 3.14159265-2.6-1.40.21 (TeX Live 2020)
+kpathsea version 6.3.2
+Copyright 2020 Han The Thanh (pdfTeX) et al.
 There is NO warranty.  Redistribution of this software is
 covered by the terms of both the pdfTeX copyright and
 the Lesser GNU General Public License.
 For more information about these matters, see the file
 named COPYING and the pdfTeX source.
 Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
-Compiled with libpng 1.6.34; using libpng 1.6.34
+Compiled with libpng 1.6.37; using libpng 1.6.37
 Compiled with zlib 1.2.11; using zlib 1.2.11
-Compiled with xpdf version 4.00
+Compiled with xpdf version 4.02
+```
+
+The above is all we need to have LaTeX work with R Markdown documents, however for Jupyter we need to add several more packages. Before we do this, please sign out of Windows and back in again in order for the Git Bash terminal to be able to find the location of TinyTex.
+
+When you sign back in,
```

Comment:
There might be a way to add it manually, but I like that it happens automatically when signing in and out.

Reply:
This seems fine.